### PR TITLE
New pull request MathExp-kpetrov2

### PR DESCRIPTION
**Pull request by kpetrov2**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by kpetrov2** [comment]
Hi @yangc95 ! I've changed the LaTex code line with an image.
I have set my username to be public and added my GitHub profile, but I think the problem persists. Should I make any more changes? I attach a picture to see my configuration.
contributor issue

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md [hunk 1]
```diff
@@ -17,9 +17,7 @@ The **`Math.Exp()`** method returns the value of raising `e` to the power of a g
 
 `e` stands for Euler's number, an irrational number, with an approximate constant value of 2.71828. It provides a constant base to the natural logarithm or log and can be understood as a limit of a given progression.
 
-$$
-\sum\limits_{n=0}^{\infty} \frac{1}{n!} = 1 + \frac{1}{n+1} + \frac{1}{(n+1)(n+2)} + \frac{1}{(n+1)(n+2)(n+3)} + \cdots
-$$
+![eFormula](https://user-images.githubusercontent.com/72105079/219855529-5cb30211-ab0d-45ca-bd19-25c36386f83c.jpg)
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:20]: The image file needs to be hosted in the [/media folder](https://github.com/Codecademy/docs/tree/main/media).
Use the following url to link to it: https://raw.githubusercontent.com/Codecademy/docs/main/media/image-name.jpg.
Change the name of the image to something like "e-formula.jpg"

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kpetrov2 I reviewed this and you need to make a few changes.

**Comment by SSwiniarski** [comment]
@kpetrov2, you may have to close this PR and start a new one making sure you're using your Github user id. The CLA is seeing an extra user, awgraves, attached to this PR.

**Pull request closed without merging**